contracts/deposit-contract/depositContract.v.py

```diff
@@ -0,0 +1,67 @@
+## compiled with v0.1.0-beta.7 ##
+
+MIN_DEPOSIT_AMOUNT: constant(uint256) = 1000000000  # Gwei
+MAX_DEPOSIT_AMOUNT: constant(uint256) = 32000000000  # Gwei
+DEPOSIT_CONTRACT_TREE_DEPTH: constant(uint256) = 32
+TWO_TO_POWER_OF_TREE_DEPTH: constant(uint256) = 4294967296  # 2**32
+SECONDS_PER_DAY: constant(uint256) = 86400
+
+Deposit: event({previous_deposit_root: bytes32, data: bytes[2064], merkle_tree_index: bytes[8]})
+ChainStart: event({deposit_root: bytes32, time: bytes[8]})
+
+CHAIN_START_FULL_DEPOSIT_THRESHOLD: uint256
+deposit_tree: map(uint256, bytes32)
+deposit_count: uint256
+full_deposit_count: uint256
+
+@public
+def __init__(depositThreshold: uint256):
```

Comment:
Can we also set the deposit size? 

For example, in the testnet, I would like to set this to 32gwei rather than 32 ether so that it isn't so "expensive" to test validator deposits.

Reply:
Alright set in a deposit size argument